tools/classifier/src/config/properties.json

```diff
@@ -2,7 +2,7 @@
   "schemaVersion": "0.0.1",
   "displayName": "File Classifier",
   "functionName": "classify",
-  "toolVersion": "0.0.36",
+  "toolVersion": "0.0.37",
```

Comment:
@jagadeeswaran-zipstack Did we take build for this ? Couldn't locate this in action runs.
Please verify the versions.

Reply:
Once this PR got merge then only I need to take build for tool right?

Reply:
@athul-rs It is always better to build after the changes are merged. Else this is a possibility that builds might get overridden by some other change made in another PR and version will be the same.

Reply:
Yes @jagadeeswaran-zipstack @athul-rs. It's better to build the tool from main itself to avoid conflicts if another person is working on it as well.
If we're sure that there's going to be no conflicts - we can take the build from the feature branch itself (after pulling any changes from `main`)